lib/serializer/CMakeLists.txt

```diff
@@ -0,0 +1,23 @@
+afr_module(serializer)
```

Comment:
Is this module supposed to be used by end user?

Reply:
Yes, it may be.